openmc/region.py

```diff
@@ -46,6 +46,25 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def get_surfaces_from_region(self, surfaces = OrderedDict()):
```

Comment:
One more thing here -- this is a method of `Region` so `_from_region` is a bit redundant. `get_surfaces` should suffice.